pkg/metrics/policystatemetrics/policystatemetrics.go

```diff
@@ -85,3 +87,47 @@ func (c *policyStateCollector) Collect(ch chan<- prometheus.Metric) {
 		strings.TrimPrefix(strings.ToLower(tetragon.TracingPolicyState_TP_STATE_ENABLED.String()), "tp_state_"),
 	)
 }
+
+// policyStateZeroCollector implements prometheus.Collector. It collects "zero"
+// metrics. It's intended to be used when the sensor manager doesn't exist, but
+// we still want Prometheus metrics to be exposed.
```

Comment:
Changes in this file looks good to me, it's a nice way to populate those custom collector metrics, thanks! Maybe a nit would have been to mention that it's used for the metrics docs in this comment, but it's pretty obvious with the `InitMetricsForDocs` above and the git commit message so it's already good.